tests/test_style.py

```diff
@@ -52,22 +53,46 @@ def test_astyle(mocker):
 
 def test_autopep8():
     styler = zazu.plugins.autopep8_styler.Autopep8Styler()
-    ret = styler.style_string('def foo ():\n  pass')
+    ret = styler.style_string('def foo ():\n  pass', None)
     assert ret == 'def foo():\n    pass\n'
     assert ['*.py'] == styler.default_extensions()
 
 
 def test_docformatter():
     styler = zazu.plugins.docformatter_styler.DocformatterStyler()
-    ret = styler.style_string('def foo ():\n"""doc"""\n  pass')
+    ret = styler.style_string('def foo ():\n"""doc"""\n  pass', None)
     assert ret == 'def foo ():\n"""doc"""\n  pass'
     assert ['*.py'] == styler.default_extensions()
 
 
+def test_eslint(mocker):
+    class MockPopen(object):
+        def __init__(self):
+            pass
+
+        def communicate(self, input=None):
+            pass
+
+        def returncode(self):
+            pass
+
+    mock_popen = MockPopen()
+    mocker.patch.object(MockPopen, 'communicate', return_value=('[{"output":"bar"}]', None))
+    mocker.patch('subprocess.Popen', return_value=mock_popen)
+    styler = zazu.plugins.eslint_styler.ESLintStyler(options=['--color'])
+    ret = styler.style_string('foo', '')
```

Comment:
Recommend using a non-empty filepath to be sure it gets passed to eslint.

Reply:
✔️